app/components/trainees/assessment_details/view.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= render_component SummaryCard::View.new(
+  title: "Assessment details",
+  heading_level: 2,
+  rows: [
+    {
+      key: "Assessment outcome",
+      value: assessment_outcome,
+    },
+    {
+      key: "Assessment end date",
```

Comment:
On the non-'dev-ref' prototype this is called "Date standards met":

![Screenshot 2020-12-03 at 09 08 55](https://user-images.githubusercontent.com/18436946/100988081-34387f00-3547-11eb-9391-0e7793f7fa50.png)

The whole component is also full-width now (whilst the title 'Check outcome details remains 2/3rds) - I double checked with Ed that this is correct.

Reply:
fixed https://github.com/DFE-Digital/register-trainee-teachers/pull/235/commits/9672f12c607a2789fff0ea03eabc16d87a320b7d